.changeset/tame-garlics-shop.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
You shouldn't use an empty changeset here; this is a change that requires a package release with a bump.